easybuild/easyconfigs/a/ASAP3/ASAP3-3.10.7-foss-2016b-Python-3.5.2.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = "PythonPackage"
+
+name = 'ASAP3'
+version = '3.10.7'
+
+homepage = 'https://wiki.fysik.dtu.dk/asap/'
+description = """ASAP is a calculator for doing large-scale classical molecular
+dynamics within the Campos Atomic Simulation Environment (ASE)."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['20d721f3eeac6982132eda91d9309baaea03f6ddeab418a0744879daf353c773']
+
+python = 'Python'
+pythonver = '3.5.2'
+versionsuffix = '-%s-%s' % (python, pythonver)
+
+ase = 'ASE'
+asever = '3.15.0'
```

Comment:
no need for these either, only used in one place, just include the values directly in `dependencies` like you should for `Python`